### DISCUSSION TITLE
Can't run next on M1

### DISCUSSION THREAD

**Discussion by Jezda1337** [vercel/next.js#40864] [Help]
I’m using mac book pro m1
node 18.9.1

EDIT:
I try the same steps on my Linux machine with the same node version, and it works fine.

I run the command npx create-next-app@latest
which init the project and when I try to run npm run dev
I get this error(image below)

<img width="664" alt="Screenshot 2022-09-24 at 02 49 34" src="https://user-images.githubusercontent.com/42359294/192073014-b93cf036-06c6-43d1-b9d0-704b951f604a.png">

After I run npm i -g next and then run the same command again npm run dev

![Screenshot 2022-09-24 at 02 50 55](https://user-images.githubusercontent.com/42359294/192073059-fdedf20f-2c80-4c21-b7f2-93845f0fe546.png)

Deleting node_modules and .next folders didn’t help.
Any help? I also downgraded the node version from 18 to 16, which didn’t help!

**Comment by ijjk** [comment]
Hi, could you share the output from `npx next info` inside your created project directory?

**Reply by Jezda1337** [reply]
Sure, thanks for the fast res 👍 
<img width="1723" alt="Screenshot 2022-09-24 at 03 00 14" src="https://user-images.githubusercontent.com/42359294/192073541-27407d42-2d5f-40db-bb9c-cb56336e575c.png">

**Reply by Markos-Th09** [reply]
Have you run `npm install` in the project?

**Reply by Jezda1337** [reply]
Yes I did, it didn't help, and here is my package.json file 
<img width="620" alt="Screenshot 2022-09-24 at 10 02 19" src="https://user-images.githubusercontent.com/42359294/192087474-1a305ad9-dd2a-400e-a27e-0550b8deac9d.png">

**Reply by Markos-Th09** [reply]
Do you use useContext in any component

**Reply by Jezda1337** [reply]
I didn't start with development, I just use the command for init the project and try to run the server, that's all :)

**Reply by Markos-Th09** [reply]
> I didn't start with development, I just use the command for init the project and try to run the server, that's all :)

Ok then it is an internal context

**Reply by Jezda1337** [reply]
Hey, after spending too much time trying to fix this I decided to try using Remix instead Next Js. Of course, that didn’t work for the same reason as next. When I try to spin Remix dev env it gives me the same message as in next (remix command is not found), then I completely reinstall my mac os system, then I set the same dev env like before, trying next and it’s work, I don’t know what the problem was could be some packages or some missing folders/files/packages, but I didn’t have time to investigate more than this, it’s not a solution, but I didn’t have time to investigate more.

**Reply by Jezda1337** [reply]
Thanks guys for the fast response and for trying to help!!
